mobile_verifier/migrations/12_disable_complete_data_checks_until.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO
+   meta
+VALUES
+   ('disable_complete_data_checks_until', '0')
```

Comment:
i didn't see this getting updated; is this only intended to be manually modified by a human operator?